src/android/FingerprintAuth.java

```diff
@@ -77,8 +79,7 @@
         AVAILABILITY,
         ENCRYPT,
         DECRYPT,
-        DELETE,
-        DISMISS
```

Comment:
Hallo @kodakjerec why DISMISS should be remove?

Reply:
I got the impression that FingerprintManagerCompat doesn't have DISMISS method